src/renderer/index.css

```
@@ -803,3 +803,252 @@
+  background: transparent;
+  color: hsl(var(--foreground));
+}
+
+.markdown-editor .bn-editor {
+  padding-top: 2.25rem;
+  padding-bottom: 1rem;
+  border-radius: 0;
+}
+
+.markdown-editor .bn-default-styles {
+  font-family: "Inter Variable", "Inter", system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)